cpp/inc/bond/ext/grpc/bond_utils.h

```diff
@@ -86,11 +86,11 @@ namespace grpc {
             {
```

Comment:
Change to: `for (grpc_slice s; grpc_byte_buffer_reader_next(&reader, &s) != 0; )`